sources/packages/web/src/constants/search-types.ts

```diff
@@ -0,0 +1,4 @@
+export enum SearchTypes {
```

Comment:
I believe this enum is not required if we follow the typical component and view approach as per https://github.com/bcgov/SIMS/pull/1857/files#r1158939706